app/views/schedule_rules/_schedule_rule_fields.html.haml

```diff
@@ -15,10 +15,13 @@
       = f.label :end_time
       = time_select24(f, :end, hours: (0..24), minute_step: @product.reserve_interval)
 
-= f.input :discount_percent, label: false, hint: false do
-  = f.label :discount_percent
-  = f.hint :discount_percent
-  = f.number_field :discount_percent, size: 4
+%h3 Global Price Group Discounts
```

Comment:
i18n please

Reply:
This should be either "Discounts" or "Price Group Discounts"